static/src/js/selectors/preferences.js

```diff
@@ -46,3 +47,21 @@ export const getGranuleSortPreference = createSelector(
     return granuleSort
   }
 )
+
+/**
+ * Retrieve the collection sort key from Redux
+ * If set to the default of usage, return null
+ * This function is used for inserting the sortKey
+ * into the URL query parameters when a user selects
+ * a different collection sorting key.
+ * @param {Object} state Current state of Redux
+ */
+export const getNondefaultCollectionSortKey = (state) => {
+  const { collectionSearchResultsSortKey } = getApplicationConfig()
+
+  const { query } = state
+  const { collection } = query
+  const { sortKey } = collection
+
+  return sortKey[0] !== collectionSearchResultsSortKey ? sortKey[0] : null
```

Comment:
```suggestion
const { collectionSearchResultsSortKey: defaultSortKey } = getApplicationConfig()

  const { query } = state
  const { collection } = query
  const { sortKey } = collection
  const [collectionSortKey] = sortKey

  return collectionSortKey !== defaultSortKey ? collectionSortKey : null
```
Just my preference I don't think we are perfectly consistent on using the es6 arr destructure

Reply:
Updated